flink/metadata.csv

```diff
@@ -0,0 +1,89 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+flink.jobmanager.Status.JVM.CPU.load,gauge,,percent,,The recent CPU usage of the JVM,0,flink,jobmanager cpu load
+flink.jobmanager.Status.JVM.CPU.time,gauge,,,,The CPU time used by the JVM,0,flink,jobmanager cpu time
+flink.taskmanager.Status.JVM.CPU.load,gauge,,percent,,The recent CPU usage of the JVM,0,flink,taskmanager cpu load
+flink.taskmanager.Status.JVM.CPU.time,gauge,,,,The CPU time used by the JVM,0,flink,taskmanager cpu time
```

Comment:
for `time` metrics, do we know what unit they are? millisecond or second?

Reply:
I couldn't find anything in the docs about it, but I will double check and also look into what values I get for these metrics.

Reply:
I _think_ these are seconds? They show up as values like 33.56G